keras_cv/models/legacy/object_detection/faster_rcnn/faster_rcnn.py

```diff
@@ -53,10 +53,10 @@ def __init__(self, **kwargs):
         self.upsample_2x = keras.layers.UpSampling2D(2)
 
     def call(self, inputs, training=None):
-        c2_output = inputs[2]
-        c3_output = inputs[3]
-        c4_output = inputs[4]
-        c5_output = inputs[5]
+        c2_output = inputs["P2"]
+        c3_output = inputs["P3"]
+        c4_output = inputs["P4"]
+        c5_output = inputs["P5"]
 
         c6_output = self.conv_c6_pool(c5_output)
         p6_output = c6_output
```

Comment:
[Leaving a comment here since Github isn't allowing me to make one at the actual lines but I'm referring to lines 78-84]
I could not change the output of this function to use the new string naming convention since it requires some changes to the following:
- Anchor Generator in lines 293 and 296
- RoI Aligner - https://github.com/keras-team/keras-cv/blob/9d62238f7b99f18eaa1d83727e4dd8d5dd5a920a/keras_cv/layers/object_detection/roi_align.py#L195-L204

Reply:
I think it would be best to update this + the ROIAligner so that we have consistency across the package.

If you'd like to also update the RetinaNet FPN, feel free to update that one as well (which currently outputs a tuple instead of a dict)